src/main/java/de/nplay/moderationbot/test/TestCommands.java

```diff
@@ -1,13 +1,29 @@
 package de.nplay.moderationbot.test;
 
 import com.github.kaktushose.jda.commands.annotations.interactions.Interaction;
+import com.github.kaktushose.jda.commands.annotations.interactions.Permissions;
 import com.github.kaktushose.jda.commands.annotations.interactions.SlashCommand;
 import com.github.kaktushose.jda.commands.dispatching.interactions.commands.CommandEvent;
+import de.nplay.moderationbot.moderation.ModerationService;
+import de.nplay.moderationbot.permissions.BotPermissionFlags;
+import net.dv8tion.jda.api.entities.Member;
+
 
 @Interaction
 public class TestCommands {
+
     @SlashCommand(value = "test ping", desc = "Testet die Erreichbarkeit des Bots.")
+    @Permissions(BotPermissionFlags.MODERATION_READ)
     public void onPing(CommandEvent event) {
-        event.reply("Test bestanden!");
+        event.reply("Täst bestanden!");
```

Comment:
typo

Reply:
intentional typo because I had some UTF-8 issues

Reply:
OK, then we can just leave it there, I guess.

Reply:
I'll remove it for you